draft-ietf-privacypass-auth-scheme.md

```diff
@@ -458,6 +459,16 @@ Template:
 * Nid: The length of the token key identifier
 * Reference: Where this algorithm is defined
 
+New entries in this registry are subject to the Specification Required registration policy
+({{!RFC8126, Section 4.6}}). Designated experts need to ensure that the token type
+is sufficiently clearly defined to be used for both token issuance and redemption,
+and meets the common security requirements for issuance protocols defined
+in {{Section 3.2 of ARCHITECTURE}}.
+
+This registry also will also allow provisional registrations to allow for experimentation
+with protocols being developed. Designated experts review and approve provisional
+registrations.
+
```

Comment:
@chris-wood please check this new text I added based on the IETF meeting discussion

Reply:
So the idea is to leave the experimental range (and duration) to the experts? If so, that works for me. I suggested a small edit that allows them to revoke provisional status so that we don't exhaust the space with experimental values.

Reply:
I don't think it's necessarily a strict experimental range, but rather allows for early allocation that then can expire and get replaced eventually.